cypress/integration/pages/frontPage/tests.js

```diff
@@ -92,6 +94,38 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
           });
         });
 
+        if (
+          serviceHasPublishedPromo(service) &&
```

Comment:
Isn't it simpler to just do `service ==='persian'`? Instead of creating a function we only use once.

Reply:
I think the idea here is to have more services with published promos in the future

Reply:
yeah this is temporary once we have services with published promos the conditional statement would be removed.